plugins/outputs/timestream/README.md

```diff
@@ -57,76 +57,50 @@ API endpoint. In the following order the plugin will attempt to authenticate.
   ## and therefore the Telegraf agent will not start.
   describe_database_on_start = false
 
-  ## The mapping mode specifies how Telegraf records are represented in Timestream.
+  ## Specifies how the data is organized in Timestream.
   ## Valid values are: single-table, multi-table.
-  ## For example, consider the following data in line protocol format:
-  ## weather,location=us-midwest,season=summer temperature=82,humidity=71 1465839830100400200
-  ## airquality,location=us-west no2=5,pm25=16 1465839830100400200
-  ## where weather and airquality are the measurement names, location and season are tags,
-  ## and temperature, humidity, no2, pm25 are fields.
-  ## In multi-table mode:
-  ##  - first line will be ingested to table named weather
-  ##  - second line will be ingested to table named airquality
-  ##  - the tags will be represented as dimensions
-  ##  - first table (weather) will have two records:
-  ##      one with measurement name equals to temperature,
-  ##      another with measurement name equals to humidity
-  ##  - second table (airquality) will have two records:
-  ##      one with measurement name equals to no2,
-  ##      another with measurement name equals to pm25
-  ##  - the Timestream tables from the example will look like this:
-  ##      TABLE "weather":
-  ##        time | location | season | measure_name | measure_value::bigint
-  ##        2016-06-13 17:43:50 | us-midwest | summer | temperature | 82
-  ##        2016-06-13 17:43:50 | us-midwest | summer | humidity | 71
-  ##      TABLE "airquality":
-  ##        time | location | measure_name | measure_value::bigint
-  ##        2016-06-13 17:43:50 | us-west | no2 | 5
-  ##        2016-06-13 17:43:50 | us-west | pm25 | 16
```

Comment:
All the changes in this section should happen in `sample.conf`. Afterwards, please run `make docs` to update the README.md file with your changes to the `sample.conf` file.

Reply:
Thanks @[srebhan](https://github.com/srebhan), I've updated sample.conf.